config/config.go

```diff
@@ -33,7 +33,7 @@ const (
 	defaultRootfsPath  = defaultFilesPath + "default-rootfs.img"
 	defaultCPUCount    = 1
 	defaultCPUTemplate = models.CPUTemplateT2
-	defaultMemSizeMb   = 128
+	defaultShimBaseDir = "/var/run/firecracker-containerd/"
```

Comment:
Debating if we should choose a different path that works on both RHEL and debian systems. Thoughts @nmeyerhans?

Reply:
Given that we're copying binaries and filesystem images to subdirectories under this path, I think we should probably change the default to something that is more likely to be backed by actual disk rather than RAM. So `/var/lib/firecracker-container/shim-base` or something like that, by default, might be OK. It reduces the likelihood that we're going to exhaust system RAM, and mount options aren't likely to vary across distros. (Mounting a separate filesystem on `/var` isn't super common these days, though it once was, whereas `/run` is a separate mount point by default on most modern distros.)

If you do make this change, double check the docs/ subdirectory to make sure we don't have references that need to be updated.